ACKNOWLEDGEMENTS.md

```diff
@@ -1,4 +1,4 @@
-Acknowlegements
+Acknowledgements
 ===============
```

Comment:
You need to fix the underlining as well.

Reply:
Fixed !